src/schemas/account_schema.py

```diff
@@ -0,0 +1,15 @@
+"""Contains the schema to easily serialize the account entity."""
```

Comment:
Why do we need these in addition to the model classes?

Reply:
@prashbnair These take of serialization, de-serialization and in turn schema validations of the JSON objects. https://flask-marshmallow.readthedocs.io/en/latest/